Neuro_Annotations.json

```diff
@@ -89,6 +89,9 @@
     "sampleName": [
         
     ],
+    "individual": [
```

Comment:
we decided to use individual, but individualID makes more sense I thin in terms of having and individualSource. Let's change it to individualID